<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SjakkWindow</class>
 <widget class="QMainWindow" name="SjakkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SjakkWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QSplitter" name="gameSplitter">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="gameTabs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTabWidget" name="infoTabs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="consoleTab">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <attribute name="title">
         <string>Console</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="outputConsole">
           <property name="font">
            <font>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="seekersTab">
        <attribute name="title">
         <string>Seekers</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="seekersTable">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gamesTab">
        <attribute name="title">
         <string>Games</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="gamesTable">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="playersTab">
        <attribute name="title">
         <string>Current Players</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="playersTable">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="HistoryLineEdit" name="cmdLine">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">margin-right: 3;
</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSjakk">
    <property name="title">
     <string>Sjakk</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionAbort"/>
    <addaction name="actionAdjourn"/>
    <addaction name="actionDraw"/>
    <addaction name="actionResign"/>
    <addaction name="separator"/>
    <addaction name="actionFlip_Board"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Sjakk"/>
   </widget>
   <addaction name="menuSjakk"/>
   <addaction name="menuGame"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="separator"/>
   <addaction name="actionSeekGame"/>
   <addaction name="actionSeek1"/>
   <addaction name="actionSeek3"/>
   <addaction name="actionSeek15"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="gameToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: white;
padding-left: 4;</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbort"/>
   <addaction name="actionDraw"/>
   <addaction name="actionResign"/>
   <addaction name="actionAdjourn"/>
  </widget>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="media.qrc">
     <normaloff>:/media/icons/Freepik/plug.svg</normaloff>:/media/icons/Freepik/plug.svg</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbort">
   <property name="icon">
    <iconset resource="media.qrc">
     <normaloff>:/media/icons/Maxim Basinski/cancel.svg</normaloff>:/media/icons/Maxim Basinski/cancel.svg</iconset>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </action>
  <action name="actionAdjourn">
   <property name="icon">
    <iconset resource="media.qrc">
     <normaloff>:/media/icons/Freepik/time-out.svg</normaloff>:/media/icons/Freepik/time-out.svg</iconset>
   </property>
   <property name="text">
    <string>Adjourn</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="icon">
    <iconset resource="media.qrc">
     <normaloff>:/media/icons/Good Ware/tie.svg</normaloff>:/media/icons/Good Ware/tie.svg</iconset>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </action>
  <action name="actionResign">
   <property name="icon">
    <iconset resource="media.qrc">
     <normaloff>:/media/icons/Smashicons/checkmate.svg</normaloff>:/media/icons/Smashicons/checkmate.svg</iconset>
   </property>
   <property name="text">
    <string>Resign</string>
   </property>
  </action>
  <action name="actionFlip_Board">
   <property name="icon">
    <iconset resource="media.qrc">
     <normaloff>:/media/icons/Vaadin/turn.svg</normaloff>:/media/icons/Vaadin/turn.svg</iconset>
   </property>
   <property name="text">
    <string>Flip Board</string>
   </property>
  </action>
  <action name="actionAbout_Sjakk">
   <property name="text">
    <string>About Sjakk</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Contents</string>
   </property>
   <property name="toolTip">
    <string>Open the help file</string>
   </property>
  </action>
  <action name="actionSeekGame">
   <property name="icon">
    <iconset resource="media.qrc">
     <normaloff>:/media/icons/seek-ad.svg</normaloff>:/media/icons/seek-ad.svg</iconset>
   </property>
   <property name="text">
    <string>Seek Game</string>
   </property>
   <property name="toolTip">
    <string>Send a seek advertisement</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionSeek1">
   <property name="icon">
    <iconset resource="media.qrc">
     <normaloff>:/media/icons/seek-ad-1.svg</normaloff>:/media/icons/seek-ad-1.svg</iconset>
   </property>
   <property name="text">
    <string>Seek 1</string>
   </property>
   <property name="toolTip">
    <string>Request 1 minute game</string>
   </property>
  </action>
  <action name="actionSeek3">
   <property name="icon">
    <iconset resource="media.qrc">
     <normaloff>:/media/icons/seek-ad-3.svg</normaloff>:/media/icons/seek-ad-3.svg</iconset>
   </property>
   <property name="text">
    <string>Seek 3</string>
   </property>
   <property name="toolTip">
    <string>Request 3 minute game</string>
   </property>
  </action>
  <action name="actionSeek15">
   <property name="icon">
    <iconset resource="media.qrc">
     <normaloff>:/media/icons/seek-ad-15.svg</normaloff>:/media/icons/seek-ad-15.svg</iconset>
   </property>
   <property name="text">
    <string>Seek 15</string>
   </property>
   <property name="toolTip">
    <string>Request 15 minute game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HistoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">historylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="media.qrc"/>
 </resources>
 <connections/>
</ui>
